### ISSUE TITLE
Can not use '/usr/bin/kubectl create -f apache.json' when follow the Configuring kubernetes on Fedora via Ansible.

### ISSUE THREAD

**Issue by BlueShells** [GoogleCloudPlatform/kubernetes#5357]
I followed the Configuring kubernetes on Fedora via Ansible.
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/getting-started-guides/fedora/fedora_ansible_config.md
It seems that need some update ,some files just changed there location....
here is the output of 'ansible-playbook -i inventory setup.yml'

******

TASK: [minion | Enable kubelet] *********************************************** 
changed: [192.168.0.19]
changed: [192.168.0.188]

TASK: [minion | Enable proxy] ************************************************* 
ok: [192.168.0.19]
ok: [192.168.0.188]

TASK: [minion | disable firewalld] ******************************************** 
ok: [192.168.0.188]
ok: [192.168.0.19]

TASK: [minion | Get iptables rules] ******************************************* 
skipping: [192.168.0.188]
skipping: [192.168.0.19]

TASK: [minion | Enable iptables at boot] ************************************** 
skipping: [192.168.0.188]
skipping: [192.168.0.19]

TASK: [minion | Open kubelet port with iptables] ****************************** 
skipping: [192.168.0.188]
skipping: [192.168.0.19]

TASK: [minion | Save iptables rules] ****************************************** 
skipping: [192.168.0.188]
skipping: [192.168.0.19]

PLAY RECAP ******************************************************************** 
192.168.0.187              : ok=27   changed=1    unreachable=0    failed=0   
192.168.0.188              : ok=18   changed=1    unreachable=0    failed=0   
192.168.0.19               : ok=18   changed=1    unreachable=0    failed=0

and then,I tried 'systemctl | grep -i kube'

● kube-apiserver.service                                                                    loaded failed failed    Kubernetes API Server
  kube-controller-manager.service                                                           loaded active running   Kubernetes Controller Manager
  kube-scheduler.service                                                                    loaded active running   Kubernetes Scheduler Plugin
● kubelet.service                                                                           loaded failed failed    Kubernetes Kubelet Server

I did not know why the apiserver loaded failed,when I run 'systemctl status apiserver'

● apiserver.service
   Loaded: not-found (Reason: No such file or directory)
   Active: inactive (dead)

and I got a message when I tried to run '/usr/bin/kubectl create -f apache.json'

F0312 13:46:50.532726   13531 create.go:59] Post http://localhost:8080/api/v1beta1/pods?namespace=default: dial tcp 127.0.0.1:8080: connection refused

did I miss some steps ?

**Comment by eparis** [comment]
clearly the problem is the apiserver not lauching.  Can you let me know what version of kube you have installed, the contents of /etc/kubernetes/* and the full output of journalctl -b -u kube-apiserver ?

**Comment by BlueShells** [comment]
thans for answer me ,I just follow the guide,and I do not know wich step I installed the kube,
I just git from the github and config the ssh,config the netwok, and run the 'ansible-playbook -i inventory setup.yml',I did not use yum to install kube ,if it installed,I think it is the command 'ansible-playbook -i inventory setup.yml' do the install.
[root@fedora ~]# ls -l  /etc/kubernetes/
total 24
-rw-r--r--. 1 root root 540 Mar 12 11:16 apiserver
-rw-r--r--. 1 root root 756 Mar 12 11:16 config
-rw-r--r--. 1 root root 224 Mar 12 11:16 controller-manager
-rw-r--r--. 1 root root 377 Dec  2 23:04 kubelet
-rw-r--r--. 1 root root 103 Dec  2 23:04 proxy
-rw-r--r--. 1 root root 111 Dec  2 23:04 scheduler

journalctl -b -u kube-apiserver
Mar 12 13:35:54 fedora kube-apiserver[11202]: F0312 13:35:54.786768   11202 apiserver.go:95] No -portal_net specified
Mar 12 13:35:54 fedora systemd[1]: kube-apiserver.service: main process exited, code=exited, status=255/n/a
Mar 12 13:35:54 fedora systemd[1]: Unit kube-apiserver.service entered failed state.
Mar 12 13:35:54 fedora systemd[1]: kube-apiserver.service failed.
Mar 12 13:35:55 fedora systemd[1]: start request repeated too quickly for kube-apiserver.service
Mar 12 13:35:55 fedora systemd[1]: Failed to start Kubernetes API Server.
Mar 12 13:35:55 fedora systemd[1]: Unit kube-apiserver.service entered failed state.
Mar 12 13:35:55 fedora systemd[1]: kube-apiserver.service failed.
Mar 12 13:43:33 fedora kube-apiserver[12749]: F0312 13:43:33.213334   12749 apiserver.go:95] No -portal_net specified
Mar 12 13:43:33 fedora systemd[1]: kube-apiserver.service: main process exited, code=exited, status=255/n/a
Mar 12 13:43:33 fedora systemd[1]: Unit kube-apiserver.service entered failed state.
Mar 12 13:43:33 fedora systemd[1]: kube-apiserver.service failed.
Mar 12 13:43:33 fedora kube-apiserver[12771]: F0312 13:43:33.535786   12771 apiserver.go:95] No -portal_net specified
Mar 12 13:43:33 fedora systemd[1]: kube-apiserver.service: main process exited, code=exited, status=255/n/a
Mar 12 13:43:33 fedora systemd[1]: Unit kube-apiserver.service entered failed state.
Mar 12 13:43:33 fedora systemd[1]: kube-apiserver.service failed.
Mar 12 13:43:33 fedora kube-apiserver[12786]: F0312 13:43:33.696629   12786 apiserver.go:95] No -portal_net specified
Mar 12 13:43:33 fedora systemd[1]: kube-apiserver.service: main process exited, code=exited, status=255/n/a
Mar 12 13:43:33 fedora systemd[1]: Unit kube-apiserver.service entered failed state.
Mar 12 13:43:33 fedora systemd[1]: kube-apiserver.service failed.
Mar 12 13:43:34 fedora kube-apiserver[12807]: F0312 13:43:34.048001   12807 apiserver.go:95] No -portal_net specified
Mar 12 13:43:34 fedora systemd[1]: kube-apiserver.service: main process exited, code=exited, status=255/n/a
Mar 12 13:43:34 fedora systemd[1]: Unit kube-apiserver.service entered failed state.
Mar 12 13:43:34 fedora systemd[1]: kube-apiserver.service failed.
Mar 12 13:43:34 fedora kube-apiserver[12823]: F0312 13:43:34.286800   12823 apiserver.go:95] No -portal_net specified
Mar 12 13:43:34 fedora systemd[1]: kube-apiserver.service: main process exited, code=exited, status=255/n/a
Mar 12 13:43:34 fedora systemd[1]: Unit kube-apiserver.service entered failed state.
Mar 12 13:43:34 fedora systemd[1]: kube-apiserver.service failed.
Mar 12 13:43:34 fedora systemd[1]: start request repeated too quickly for kube-apiserver.service
Mar 12 13:43:34 fedora systemd[1]: Failed to start Kubernetes API Server.
Mar 12 13:43:34 fedora systemd[1]: Unit kube-apiserver.service entered failed state.
Mar 12 13:43:34 fedora systemd[1]: kube-apiserver.service failed.
